.github/workflows/cli.yml

```diff
@@ -43,6 +43,20 @@ jobs:
               - '.github/**/*'
               - '!.github/workflows/packages.yml'
 
+  prettier:
```

Comment:
Could we also have this for the packages?

Reply:
Yeah, should probably be a separate PR though since I think there's some fixes to be done there